### PR TITLE
community[patch]: support stream for wenxin and zhipu chat

### LINKED ISSUE #5567: ChatBaiduWenxin stream mode doesn't work as expected
### Checked other resources

- [X] I added a very descriptive title to this issue.
- [X] I searched the LangChain.js documentation with the integrated search.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I am sure that this is a bug in LangChain.js rather than my code.
- [X] The bug is not resolved by updating to the latest stable version of LangChain (or the specific integration package).

### Example Code

```typescript     
const model = new ChatBaiduWenxin({
    model: "ERNIE-Bot", // Available models are shown above
    streaming: true,
    temperature: 1,
    baiduApiKey: BAIDU_API_KEY, // In Node.js defaults to process.env.BAIDU_API_KEY
    baiduSecretKey: BAIDU_SECRET_KEY, // In Node.js defaults to process.env.BAIDU_SECRET_KEY
});

const stream = await model.stream("Hello! Tell me about yourself.");
const chunks = [];
for await (const chunk of stream) {
   chunks.push(chunk);
   console.log(`${chunk.content}|`);
}
```

### Error Message and Stack Trace (if applicable)

Current output is return the entire final results
image
it should be streaming mode return like this
```
|
Hello|
!|
 I'm|
 an|
 AI|
 language|
 model|
 created|
 by|
 Open|
...
```

### Description

When using ChatBaiduWenxin, how to return in a streaming fashion instead of returning a whole readable stream.

### System Info

langchain 0.2.2
mac

### PR DESCRIPTION
**Pull request by jeasonnow**
Fixes #5567 #5508

### CODE REVIEW

**Comment by jacoblee93** [comment]
I'm unable to test but this looks good!

**Comment by jacoblee93** [comment]
Sorry, had a second look and we may be able to avoid creating a new util?

File: libs/langchain-community/src/utils/stream.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { IterableReadableStream } from "@langchain/core/utils/stream";
+
+export async function* createStream<T = unknown>(
```
[the remark this answers is not in the document]
Comment (jeasonnow) [reply libs/langchain-community/src/utils/stream.ts:3]: I didn't notice this utility before, I will use it to resubmit.

**Comment by jacoblee93** [comment]
Thank you!

**Pull request merged**